base_classes/NXobject.nxdl.xml

```diff
@@ -27,8 +27,41 @@
 	name="NXobject" 
 	type="group">
 	<doc>
-		This is the base object of NeXus
+        This is the base object of NeXus. The groups and fields contained 
+        within this file are allowed to be present in any derived base class.
+
+        If nameType="partial", the placeholders (e.g., FIELDNAME or GROUPNAME)
+        can be replaced by the name of any object (field or group, 
+        respectively) that exists within the same group.
 	</doc>
-        <!--attribute name="name"><doc>name of instance</doc></attribute-->
+    <group type="NXcollection" minOccurs="0" />
+    <group type="NXdata" minOccurs="0" />
+    <group type="NXlog" minOccurs="0" />
+    <group type="NXnote" minOccurs="0" />
+    <group type="NXparameters" minOccurs="0" />
+    <group name="GROUPNAME_log" type="NXlog" nameType="partial" minOccurs="0">
+        <doc>
+            NXlog group containing logged values of GROUPNAME.
+        </doc>
+    </group>
+    <field name="FIELDNAME_set" type="NX_NUMBER" nameType="partial" minOccurs="0">
+        <doc>
+            Target values of FIELDNAME. 
+        </doc>
+    </field>
+    <field name="FIELDNAME_errors" type="NX_NUMBER" nameType="partial" minOccurs="0">
+        <doc>
+            Uncertainties of FIELDNAME values.
```

Comment:
Perhaps:

Uncertainties of FIELDNAME measurements.  Uncertainty values should be one sigma for normal errors.  Twice the uncertainty value provides the 68% confidence interval.